### ISSUE TITLE
Error when initializing neosnippet cache.

### ISSUE THREAD

**Issue by mvpgomes** [spf13/spf13-vim#752]
I using vim 7.4 in OS X, when opening any file and type the `INSERT` shortcut to edit the file, I'm getting the following error:

```
Error detected while processing function neosnippet#init#_initialize..
Cannot create directory: /Users/../.cache/neosnippet
```

I think that is a the neosnippet plugin does not have permissions to write in the `.cache` directory. How this could be fixed?

**Comment by hgupta** [comment]
Try giving `.cache` directory full permission i.e. `777`. It worked for me.

**Comment by ghost** [comment]
so what's the normal user directory on OSX?  /Users/UserName ?

**Comment by mvpgomes** [comment]
Yes it is /Users/UserName. In my case the owner of the `.cache` directory was the `root`. To fix the problem I simply changed the owner of the directory to UserName.

**Comment by thexande** [comment]
@mvpgomes : your solution works. 
`sudo chown -R $USERNAME ~/.cache`

@hgupta : Your solution gives ANY user in caching directory the ability to read / write / execute. Thats a problem.
